app/components/atoms/horizontalButton/HorizontalButton.component.tsx

```diff
@@ -0,0 +1,36 @@
+import {FC} from 'react';
+
+import {
+  Arrow,
+  ButtonFace,
+  ContentText,
+  HorizontalButtonProps,
+  StyledHorizontalButton,
+} from './HorizontalButton.styles';
+
+export const HorizontalButton: FC<HorizontalButtonProps> = ({
+  content,
+  ...props
+}) => {
+  return (
+    <StyledHorizontalButton {...props}>
+      <ButtonFace>
+        <ContentText>{content}</ContentText>
+        <Arrow>
+          <svg
```

Comment:
is it possible to just create the .svg file and import it here or what do you think

Reply:
I think it's possible since I'm not really manipulating the SVG's properties, so it could just be something like `<img src="arrow.svg" />`, it's just that I kinda like to preserve the SVG and I haven't found a better solution for that.
Either way, an img should work just fine in this case